Formula/portal.rb

```diff
@@ -0,0 +1,63 @@
+class Portal < Formula
+  desc "Quick and easy command-line file transfer utility from any computer to another"
+  homepage "https://github.com/SpatiumPortae/portal"
+  url "https://github.com/SpatiumPortae/portal/archive/v1.2.2.tar.gz"
+  sha256 "94c334285925175edfc470239ea2cc73a0039f9487ce92e60764f0965aba66d4"
+  license "MIT"
+  head "https://github.com/SpatiumPortae/portal.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-s -X main.version=v#{version}"
+    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/portal/"
+  end
+
+  test do
+    # Simple version check test.
+    system "#{bin}/portal", "version"
```

Comment:
We should check the output then
```suggestion
    # Simple version check test.
    assert_match version.to_s shell_output("#{bin}/portal version")
```

Reply:
Thank you! Fixed in [53f50ed](https://github.com/Homebrew/homebrew-core/pull/125727/commits/53f50ed2a56e63d3a9f9d31e405005d3040b6b90)